background/analytics.js

```diff
@@ -0,0 +1,34 @@
+import analytics from 'universal-ga';
+
+import Config from './config';
+import { isProd } from '../shared';
+
+const trackingId = 'UA-77071647-4';
+
+Config.get()
+  .then(config => {
+    if (!config.analyticsEnabled) {
+      return;
+    }
+
+    // initialize
+    analytics.initialize(trackingId, {
+      debug: !isProd,
+      clientId: config.clientId
+    });
+    // Disable protocol check
+    analytics.set('checkProtocolTask', null);
+    analytics.set('dimension1', config.lastVersion);
+    analytics.set('dimension2', config.lastChain);
```

Comment:
It's required by analytics platform afaict: https://support.google.com/analytics/answer/2709828?hl=en#collection

More explicit names can be defined in the analytics UI.